person.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+class Person < Nameable
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', _parent_permission = 1)
+    super()
+    @age = age
+    @name = name
+    @parent_permission = 1
```

Comment:
Great job @haftamudesta 🤝

- [ ] Kindly ensure @rentals is initialized. You may use this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more guidance.